planet/src/main/webapp/js/authentication.js

```diff
@@ -24,6 +24,11 @@ const firebaseConfig = {
 };
 
 firebase.initializeApp(firebaseConfig);
+const database = firebase.database();
+database.ref('users').on('value', function(snapshot) {
+    console.log(snapshot.val());
+    updateListOfUsers(snapshot.val());
+});
```

Comment:
This looks like it's retrieving a list of all users for display. That won't scale beyond a few dozen users. What is the plan to support a larger number of users?

Reply:
This is just an example of how to retrieve data in the database. It will be removed later!